### ISSUE TITLE
Hyperparameters for finetuning on LLM-generated data

### ISSUE THREAD

**Issue by evgenyrp** [mozilla/firefox-translations-training#1346]
I tried a bunch, including the ones @ZJaume recommended, but couldn't beat our regular ones for training.

For example, these:
```yaml
  # domain adaptation params
  early-stopping: '20'
  exponential-smoothing: '0.0001'
  valid-freq: '100'
  save-freq: '100'
  disp-freq: '50'
  disp-first: '10'
  learn-rate: '6e-06'
  lr-warmup: '200'
  lr-decay-inv-sqrt: '2000'
```

Another thing is chrf is almost always decreasing after first validation. I see on the [evals dashboards](https://mozilla.github.io/translations/final-evals/?langpair=en-zh) that chrf is lower for `llmaat` fine-tuned models but neural metrics (COMET, MetricX) have improved. Maybe chrf is not a good validation metric for this case. We can try using a custom validation script ([example](https://github.com/marian-nmt/marian-examples/blob/master/training-basics/scripts/validate.sh)) that runs MetricX or COMET. The fine-tuning data is slightly biased towards MetricX because it was filtered with MetricX-QE < 2 but since the improvement correlates with COMET I think it's worth exploring. 

The hypothesis is that LLM-produced data is more fluent in a language, so lexical metrics degrade, but neural ones improve.

**Comment by ZJaume** [comment]
> I think the less improvement we have, the more lexical metrics fluctuate, so they can't be trusted for minor improvement around 1-2 COMET points.

I agree.


> From this, I conclude that it's not ideal to use chrf as a stopping criterion for finetuning.

Hope CE is still useful.
